package.json

```diff
@@ -57,6 +57,7 @@
     },
     "scripts": {
         "test": "nyc --reporter=lcov mocha tests/index.js",
-        "lint": "eslint --ext js \"**/*.js\""
+        "lint-js": "eslint --ext js \"**/*.js\"",
+        "lint-python": "python3.6 -m flake8 ./"
```

Comment:
In `python-caller-trampoline` we canonically call python with `python3`, so all our environments ensure that there is a `python3` binary that does the right thing.

Reply:
I did this because the docs for `flake8` specifically call out that reported issues are dependent on the version of python that `flake8` is being run with. Which is terrible... but I'd rather have linting be as deterministic as possible.

Reply:
Huh, fair enough